src/EditMode.jsx

```diff
@@ -0,0 +1,30 @@
+import React, { useState } from 'react';
+import EditView from './EditView';
+import DisplayView from './DisplayView';
+
+  /************************************
+   * Constants
+   ************************************/
+
+const EDIT_VIEW = 'EDIT';
+const DISPLAY_VIEW = 'DISPLAY';
+
+export default function EditMode() {
+  /************************************
+   * State
+   ************************************/
+
+  const [view, setView] = useState(EDIT_VIEW);
+
+  /************************************
+   * Render
+   ************************************/ 
+  
+  return (
+    <div>
+        <button onClick={() => {setView(EDIT_VIEW)}}>Edit</button>
+        <button onClick={() => {setView(DISPLAY_VIEW)}}>Preview</button>
```

Comment:
Embracing arrow functions I see! You should get rid of the braces around the `setView` function call.

Reply:
I consider these more of lambdas but i guess they are haha. Just trying to get basic functionality in right now, will implement fully when parsing starts

Reply:
I took care of this in my most recent PR. Not enough functionality here to care about blocking a merge for that reason lmao

Reply:
> I consider these more of lambdas 

Ditto